packages/ui/src/machines/authenticator/actions.ts

```diff
@@ -148,8 +148,9 @@ export const setUnverifiedContactMethods = assign({
       event.data as Auth.FetchUserAttributesOutput;
 
     return {
-      ...(email_verified === 'false' && { email }),
-      ...(phone_number_verified === 'false' && { phone_number }),
+      ...(email_verified === 'false' && email && { email }),
+      ...(phone_number_verified === 'false' &&
+        phone_number && { phone_number }),
```

Comment:
Is there a scenario where both line 151 and 152 resolve to meaningful values?

Reply:
There is a case, this was just the short hand way of conditionally assigning the value only if the value exists. E.g. avoiding adding `{phone_number: undefined}`